roles/dcos_cli/defaults/main.yml

```diff
@@ -10,7 +10,19 @@ dcos_cli_frameworks_list:
   - chronos
 dcos_cli_apps_list:
   - prometheus
+  - elasticsearch
+  - logstash
+  - kibana
 
 # apps
 prometheus_image: prom/prometheus
```

Comment:
* Why are we breaking the name convention here?
* (Also for prometheus) Why dont we put the app specific variables under vars/app-name.yml as we are 
```
- include_vars: "{{ item }}.yml"
  with_items:
    - "{{ dcos_cli_apps_list }}"
```